code/obj/item/device/lights.dm

```diff
@@ -113,13 +114,19 @@
 	col_b = 0.1
 	brightness = 0.5
 	height = 0.75
+	var/color_name = "green"
 
 	proc/burst()
 		var/turf/T = get_turf(src.loc)
 		make_cleanable( /obj/decal/cleanable/generic,T)
 		make_cleanable( /obj/decal/cleanable/greenglow,T)
 		qdel(src)
 
+	proc/turnon()
+		on = 1
+		icon_state = "[base_state][on]"
+		light.enable()
```

Comment:
should be simple lights, but out of scope - would require refactor of path

Reply:
yeah, itll be worth dedicating some time to do that in a separate PR